shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -373,6 +386,7 @@ public void onTouchExplorationStateChanged(boolean isTouchExplorationEnabled) {
         if (platformViewsAccessibilityDelegate != null) {
             platformViewsAccessibilityDelegate.attachAccessibilityBridge(this);
```

Comment:
Can you clarify the relationship between the PlatformViewsAccessibilityDelegate and the new AccessibilityViewEmbedder?  I think I was expecting this embedding behavior to be handled by the delegate, so now I'm a bit confused.

Reply:
In our offline discussion we concluded that the piece that takes an Android View and creates the respective a11y tree would live in the A11yBridge and not in the PlatformViewsController. AccessibilityViewEmbedded is that piece.

PlatformViewsAccessibilityDelegate is what allows the AccessibiiltyBridge to ask PlatformViewsController for a view given an ID, and allows the AccessibilityBridge to subscribe itself as the target for accessibility events originating in embedded Views.

Reply:
Ok, thanks